### DISCUSSION TITLE
MikoPBX 2021.3.77

### DISCUSSION THREAD

**Discussion by jorikfon** [mikopbx/Core#179] [General ( Обо всём )]
**Внимание!!!**
  В это версии обновлены ключевые библиотеки, версия Linux. Если вы используете какие-либо модули расширений необходимо будет зайти в личный кабинет системы лицензирования и сбросить старые привязки лицензии к хостам (HOST LOCK) по адресу https://lm.miko.ru в разделе **Монитор сессий*

  В противном случае, могут возникнуть проблемы в работе лицензируемых модулей расширений. 
Привязки надо сбросить после обновления PBX.

<hr /><em>This discussion was created from the release <a href='https://github.com/mikopbx/Core/releases/tag/2021.3.77'>MikoPBX 2021.3.77</a>.</em>

**Comment by enigmen** [comment]
Уточните пожалуйста как правильно произвести обновление? текущая версия 2019.4.53.
